holoviews/core/spaces.py

```diff
@@ -410,9 +417,12 @@ class Callable(param.Parameterized):
     inputs = param.List(default=[], doc="""
          The list of inputs the callable function is wrapping.""")
 
-    def __init__(self, **params):
+    def __init__(self, callable_function=None, stream_mapping={}, **params):
+        if callable_function is not None:
+            params['callable_function'] = callable_function
```

Comment:
Any reason ``stream_mapping`` can't just be a parameter, allowing us to go back to the ``def __init__(self, **params)`` constructor?

In addition, how can ``callable_function`` be None? Isn't it required to define a valid ``Callable``?